### ISSUE TITLE
Service request bitview.net

### ISSUE THREAD

**Issue by ItsOpenSourceSoftware** [imputnet/cobalt#849]
### service name

bitview.net

### service description

Video sharing website like youtube that lets you watch videos and upload videos.

### link samples

```shell
https://www.bitview.net/watch?v=G6VUOh2j
```

**Comment by Lekuruu** [comment]
I second this

**Comment by ItsOpenSourceSoftware** [comment]
i found this in the network tab on firefox dev tools and found this

![Image](https://github.com/user-attachments/assets/c90ebfd0-37c5-421a-94f6-0fbada786303)

![Image](https://github.com/user-attachments/assets/c20b7688-1499-4e39-a36d-4d6933b4cc96)

**Comment by Lekuruu** [comment]
Yeah, bitview is using a simple html `video` tag. Should not be too hard to parse.
